Git Formats/Git Config.sublime-syntax

```diff
@@ -7,6 +7,8 @@ name: Git Config
 file_extensions:
   - gitconfig                # /etc/gitconfig
   - .gitconfig               # ~/.gitconfig
+  - gitmodules               # /etc/gitmodules
```

Comment:
As your comment states, the .gitmodules file exists in the top-level of a working tree. Therefore a globally used file in a directory like `/etc/gitmodules` will most likely not exist, I think.

Reply:
Oh, no that doesn't make sense.